package exceptions_handling;

public class ExampleSow {

	public static void main(String[] args) {
		try {
			int a[]= {2,3,4};
			System.out.println("Array Element:"+a[1]);
			int b=20/2;
			System.out.println("Element b is:"+b);
			String c="abc";
			System.out.println("Value of c:"+c);
			int d[]= {3,4,5,6};
			System.out.println("Te length of te arrray is :"+d.length);
			
		}
		catch(Exception e) {
			
		}
		

	}

}
